src/wasm-s-parser.h

```diff
@@ -351,7 +350,6 @@ class SExpressionWasmBuilder {
       // unnamed, use an index
       func->name = Name::fromInt(functionCounter);
     }
-    if (debug) std::cerr << "parse function " << func->name << '\n';
```

Comment:
this was causing the excessive logging in debug mode. removing it forces removing the property from the class or else the compiler errors on unused property.